apps/web/src/modules/dao/components/MembersList/MembersList.tsx

```diff
@@ -0,0 +1,103 @@
+import { Flex, Text } from '@zoralabs/zord'
+import axios from 'axios'
+import { useRouter } from 'next/router'
+import React, { useMemo } from 'react'
+import useSWR from 'swr'
+
+import Pagination from 'src/components/Pagination'
+import { usePagination } from 'src/hooks/usePagination'
+import { useLayoutStore } from 'src/stores'
+import { useChainStore } from 'src/stores/useChainStore'
+
+import { useDaoStore } from '../../stores'
+import { MemberCard } from './MemberListCard'
+import { MemberCardSkeleton, MembersPanel } from './MembersListLayout'
+
+export type DaoMember = {
```

Comment:
can we reuse this type from the subgraph request file

Reply:
Absolutely!